sig-node/governance.md

```diff
@@ -0,0 +1,181 @@
+# SIG Node Governance
+
+## Roles
+
+Membership for roles tracked in: [OWNERS]
+
+- **Chairs**
+  - Run operations and processes governing the SIG
+  - *MUST* remain active in the role and are  removed from the position if they
+    are unresponsive for > 3 months and *MAY* be removed if not proactively
+    working with other chairs to fulfill responsibilities.  To remove an
+    inactive chair, a majority vote from existing chairs should occur. If a
+    [super-majority] is not possible, the [steering-committee] may sponsor
+    removal.
+  - *MAY* declare an intention to take sabbatical for no more than 3 months in a
+    calendar year.  Chairs should notify existing chairs of their intent, and
+    absent at least 2 remaining chairs, *MUST* nominate a temporary replacement
+    from the existing set of technical leads during this period.
+  - *SHOULD* represent a diverse set of organizations.  This is predicated on
+    interest in the role from multiple organizations with qualified candidates.
+    Qualification is measured by existing chairs and *SHOULD* be supported by a
+    majority of SIG Members under [lazy-consensus].
+  - *MAY* decide to step down at any time and propose a replacement.  Use lazy
+    consensus amongst chairs with fallback on majority vote to accept proposal.
+    This *SHOULD* be supported by a majority of SIG Members.
+  - *MAY* select additional chairs through a [super-majority] vote amongst
+    chairs.  This *SHOULD* be supported by a majority of SIG Members.
+  - Number: 1-3
+  - Defined in [sigs.yaml]
+
+- **Technical Leads**
```

Comment:
TL feels have overlap with Chairs, is it expected?

Reply:
yes, it is expected.  the traditional sig lead role has been split into chair and technical lead roles.  for sig-node, i anticipate there will be overlap where chairs will also serve as technical leads, but we may want to expand the list of tech leads to at least have an odd number.